### PR TITLE
change MAINTAINER to LABEL

### PR DESCRIPTION
**Pull request by ben-st**
i changed maintainer to label, since the former is deprecated.
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

All the best,
Benjamin

### CODE REVIEW

**Comment by ben-st** [comment]
I signed it!

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Prepare the image with:
 #   docker build -t tensorflow/haskell:v0 docker
 FROM tensorflow/tensorflow:1.3.0
-MAINTAINER TensorFlow authors <tensorflow-haskell@googlegroups.com>
+LABEL maintainer "TensorFlow authors <tensorflow-haskell@googlegroups.com>"
```
Comment (blackgnezdo) [inline docker/Dockerfile]: The syntax in the page you referenced is
LABEL maintainer=
please adjust.

Comment (ben-st) [reply docker/Dockerfile]: i fixed it, thx 👍

**Comment by ben-st** [comment]
i updated the 2 other files and changed to referenced syntax with the = character.

**Comment by ben-st** [comment]
@blackgnezdo i did the requested changes.
Could you please review?

All the best

**Comment by blackgnezdo** [comment]
Jenkins test this please.

**Pull request merged**